org/all-prs.ts

```diff
@@ -1,31 +1,72 @@
 import { schedule, danger, warn, fail } from "danger"
-const pr = danger.github.pr
 
 const isJest = typeof jest !== "undefined"
 
-const rfc = isJest ? require("./utils").rfc : (id: string, reason: string, closure: any) => closure()
+// Stores the parameter in a closure that can be invoked in tests.
+const storeRFC = (reason: string, closure: () => void | Promise<any>) => {
+  // We return a closure here so that the (promise is resolved|closure is invoked) 
+  // during test time and not when we call rfc().
+  return () => {
+    return closure instanceof Promise ? closure : Promise.resolve(closure())
+  }
+}
```

Comment:
I saw where you were going in #22 and played around with it a bit. There's a bit of trickery here where `storeRFC` and `runRFC` have different return types, since we're purposefully deferring execution in the test case by storing the closure invocation inside another closure. It's not idea, but I did my best to document things.